agenda.php

```diff
@@ -184,32 +195,38 @@ protected function removeDeletedEvents($etags) {
     }
 
     private function updateEvents($urls) {
-        $xml = $this->caldav_client->updateEvents($urls);
+        $events = $this->caldav_client->updateEvents($urls);
         
-        foreach($xml as $event) {
-            if(isset($event['value']['propstat']['prop']['{urn:ietf:params:xml:ns:caldav}calendar-data'])) {
-                $eventName = basename($event['value']['href']);
-                $this->log->info("Adding event " . $eventName);
-                
-                $this->localcache->deleteEvent($eventName);
-                
-                
-                // parse event to get its DTSTART
-                $vcal = \Sabre\VObject\Reader::read($event['value']['propstat']['prop']['{urn:ietf:params:xml:ns:caldav}calendar-data']);
-                $startDate = $vcal->VEVENT->DTSTART->getDateTime();
-                
-                if($startDate < new DateTime('NOW')) {
-                    $this->log->debug("Event is in the past, skiping");
-                    continue;
-                }
-                
-                $this->localcache->addEvent($eventName, $event['value']['propstat']['prop']['{urn:ietf:params:xml:ns:caldav}calendar-data'], trim($event['value']['propstat']['prop']['getetag'], '"'));
+        if(is_null($event) || $event === false) {
```

Comment:
I missed it earlier but I think there is a typo here (should be `$events` with an S)